### PR TITLE
Update UI calculator variable.

### PR DESCRIPTION
**Pull request by adamcogbill**

### CODE REVIEW

**Comment by adamcogbill** [comment]
The original request:
image

File: UI/BenefitsCalculator/src/components/Form/output.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ const Output = (props) => {
   const helpTextWeeks2Q = 'weeks in the combined quarters';
   const helpTextWeeks1Q = 'weeks in the quarter';
   const helpTextDisqualification1 = `You must have earned at least ${toCurrency(quartersSumThreshhold)} during the last 4 completed calendar quarters to be eligible.`;
-  const helpTextDisqualification2 = `Your total base period wages of ${toCurrency(quartersSum)} must be equal to or greater than ${toCurrency(weeklyBenefitFinal * 30)} (your weekly benefit amount x 30) to be eligible.`;
+  const helpTextDisqualification2 = `Your total base period wages of ${toCurrency(quartersSum)} must be equal to or greater than ${toCurrency(weeklyBenefitFinal * 26)} (your weekly benefit amount x 30) to be eligible.`;
```
Comment (clairesunstudio) [inline UI/BenefitsCalculator/src/components/Form/output.js:65]:
```suggestion
  const helpTextDisqualification2 = `Your total base period wages of ${toCurrency(quartersSum)} must be equal to or greater than ${toCurrency(weeklyBenefitFinal * 26)} (your weekly benefit amount x 26) to be eligible.`;
```

**Comment by clairesunstudio** [comment]
I also noticed that we have this disclaimer which can be confusing, should we update this message also?
Screen Shot 2021-03-26 at 10 17 52 AM

**Comment by adamcogbill** [comment]
I'll ask!

**Review by clairesunstudio — APPROVED** [review]

**Pull request merged**